docs/overview.md

```diff
@@ -0,0 +1,98 @@
+
+As part of the deployment process, you can define variables that change based on the scope of your deployments, this means, for instance, you can use different connection strings when you deploy to different environments.
+
+After the initial setup, your deployment process shouldn't change between deployments even though the software being deployed will change as part of the development process.
+
+Learn more about the [deployment process](/docs/deployment-process/index.md), how you can set up [projects](/docs/deployment-process/projects/index.md), use [variables](/docs/deployment-process/variables/index.md) and [deploy releases](/docs/deployment-process/releases/index.md).
+
+## Deploy your Software
+
+Once you have access to an Octopus Server, your infrastructure is configured, your applications packaged, and the deployment process defined, you're ready to start deploying your software. To deploy your software, you first create a release.
+
+A release is a snapshot of your deployment process, configuration variables, and software packages. Releases are created from projects and deployed to your environments.
+
+When you deploy a release, you are executing the deployment process with all the associated details, as they existed when the release was created. You can deploy a release as many times as you want to.
+
+Learn more about how [deploying releases](/docs/deployment-process/releases/index.md), or see some [deployment examples](/docs/deployment-examples/index.md).
+
+## The Delivery Pipeline
+
+A typical workflow using Octopus Deploy could something like the following:
+
+1. **Commit Code to Your Existing Source Control System.**
+
+   You might be using Git, Team Foundation Server, Subversion, or Mercurial. The choice is yours.
+
+1. **Your CI/Build Server Compiles the Code and Runs Unit Tests.**
+
+   You might be using TeamCity, Jenkins, Bamboo, Azure DevOps, Team Foundation Server (TFS), or CruiseControl.NET. Again, the choice is yours.
+
+1. **Package Your Application.**
+
+   When the build is ready, your CI/build server takes all the files your software needs to run and bundles them up ready for deployment.
+
+1. **Octopus Deploy Deploys Your Software to Your Infrastructure.**
+
+   Octopus deploys your software to the infrastructure you've configured, whether this is on-premises servers or cloud services. Because you likely want to deploy your software into a testing environment before deploying into production, Octopus promotes releases of your software through your environments, for instance, to dev, testing, staging, and production, and because each environment has slightly different configurations, Octopus manages those for you too.
```

Comment:
Consider ". For instance" to break this long sentence up.

Reply:
I've rewritten that paragraph to take some of the wordiness out of it and get to the point quicker:
   Octopus deploys your software to the infrastructure you've configured, whether that's on-premises servers or cloud services. Octopus promotes releases of your software through your environments from development, to test, and finally into production, and because each environment has slightly different configurations, Octopus manages those for you too.